.gitignore

```diff
@@ -4,7 +4,7 @@
 bin
 vendor/bundle
 *.gem
-*emfile.lock
+gemfiles/*.lock
```

Comment:
As discussed in #41, the `Gemfile.lock` in the root directory has been added. Only the lockfiles by Appraisal are ignored.